app/src/main/java/com/claraberriel/mvpparking/mvp/presenter/ParkingSizePresenter.java

```diff
@@ -0,0 +1,40 @@
+package com.claraberriel.mvpparking.mvp.presenter;
+
+import android.util.Log;
+
+import com.claraberriel.mvpparking.mvp.model.ParkingModel;
+import com.claraberriel.mvpparking.mvp.view.ParkingSizeView;
+
+public class ParkingSizePresenter {
+
+    private final ParkingModel model;
+    private final ParkingSizeView view;
+
+
+    public ParkingSizePresenter(ParkingModel model, ParkingSizeView view) {
+        this.model = model;
+        this.view = view;
+    }
+
+    //Event handlers
+    public void onSubmit() {
+        try {
+            view.showParkAmount(view.getEdittextNumber());
+        } catch (NumberFormatException e) {
+            Log.e(ParkingSizeView.class.getSimpleName(), e.toString());
+            view.showInvalidError();
+        }
+        model.setParkingSize(view.getEdittextNumber());
+        view.showParkAmount(model.getParkingSize());
+    }
+
+    /*
+    Presenter
+
+    decides what to do with user input.
+    gathers data from the model.
+    tells the View what to do.
+
+     */
```

Comment:
Move this block just before the class definition:
```
/**
 * Decides what to do with user input. Gathers data from the model. Tells the View what to do.
*/
```
This is a JavaDoc comments at class level.